rumqttc/src/v5/framed.rs

```diff
@@ -15,13 +15,13 @@ pub struct Network {
     /// Buffered reads
     read: BytesMut,
     /// Maximum packet size
-    max_incoming_size: usize,
+    max_incoming_size: Option<usize>,
```

Comment:
This shouldn't be optional. There are always default values for `max_incoming_size`. Either via config or server connack. Does the spec have any restrictions about making this an optional filed?

Reply:
Added `default_max_incoming_size` to mqtttoptions